johnsonandjohnson/Bodiless-JS
# fix(test-site): Test site needs Footer rendered with new design api #288

## Changes

- Updated Footer to use Design API
- Incorporated auto-render of date built (required in brand sites)
- Fixed link on home page bottom left tout

## Test Instructions
 - Verify footer renders on bottom of page in all pages.
 - Verify link on homepage left tout now takes you to documentation link in docs.

## Related Issues
